explainer.md

```diff
@@ -30,6 +30,16 @@ Media Feeds provides a way for user agents to discover such feeds and provides a
 -   [MediaRSS/RSS](http://www.rssboard.org/media-rss#media-content) is an alternative, however it does not give us enough detail about whether the content is a TV show, movie as well as the structure around a TV series and seasons.
 
 
+## User Journey
+
+1. User navigates to video site in order to see last episode of season #1.
+
+2. Metadata loaded via link tag includes an entry for first episode of season 2.
+
+3. User closes tab.
+
+4. The next day, the browser surfaces a suggestion on the NTP to watch the first episode of season 2. Currently, sites that want to recommend next videos do so with their own notifications; this way the UA can automatically do so across sites and with higher quality and control by the user.
```

Comment:
Is it worth saying "new tab page" and "user agent" in full here, or otherwise clarifying the acronym?

Reply:
Yes, please. I just came here to say the same thing.